lab-trent/lib/resourcemanager.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+
+const DATA_PATH = './data';
+
+let resourceCache = { };
+
+exports.addResource = function(resource) {
```

Comment:
Best not to use the exports object directly like this. You can think of it as a locally scoped variable, which has the potential to cause issues. Rather, use module.exports explicitly.